libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/MapboxRouteLineOptions.kt

```diff
@@ -277,6 +290,14 @@ class MapboxRouteLineOptions private constructor(
             return this
         }
 
+        /**
+         * Used for throttling the interval of updates to the vanishing route line.
+         * The default value is [RouteLayerConstants.DEFAULT_VANISHING_POINT_MIN_UPDATE_INTERVAL_NANO].
+         */
+        fun vanishingRouteLineUpdateIntervalNano(
+            interval: Long = RouteLayerConstants.DEFAULT_VANISHING_POINT_MIN_UPDATE_INTERVAL_NANO
+        ): Builder = apply { this.vanishingRouteLineUpdateIntervalNano = interval }
```

Comment:
```suggestion
        fun vanishingRouteLineUpdateInterval(interval: Long, unit: TimeUnit): Builder =
            apply { this.vanishingRouteLineUpdateIntervalNano = unit.toNanos(interval) }
```

Reply:
I changed the name and clarified the kdoc.

Reply:
Do users always need nanosecond precision here? I don't think it would be friendly to make them pass nanoseconds, because these numbers are usually huge. And with a second `TimeUnit` argument we get a nice readability:
```kotlin
builder.vanishingRouteLineUpdateInterval(50, TimeUnit.MILLISECONDS)
```

Reply:
The value used is in nano seconds. It's important that a developer deciding to deviate from the default value understand the effects are at the nano second level.

Reply:
I really doubt a human can notice the difference even if they change the value by thousands of nanoseconds  But we can keep it your way, user can always easily convert to nanoseconds themselves.